cli/cli_app.go

```diff
@@ -1057,7 +1070,7 @@ func providersNeedInit(terragruntOptions *options.TerragruntOptions) bool {
 //
 // If terraformSource is specified, then arguments to download the terraform source will be appended to the init command.
 //
-// This method will return an error and NOT run terraform init if the user has disabled Auto-Init
+// # This method will return an error and NOT run terraform init if the user has disabled Auto-Init
```

Comment:
Please revert all unnecessary changes to comments in this PR.

Reply:
Fixed [f334d25](https://github.com/gruntwork-io/terragrunt/pull/2362/commits/f334d25d508af32dd3c0fac2fe68450cb3fd2628) . My pre-commit locally is broken, so the quick fix was to run `goimports -l -w .`... it changed unrelated things.